setup-dnt.sh

```diff
@@ -17,7 +19,7 @@ fi
 
 if [ $# -gt 0 ] ; then
   NODE_VERSIONS=$*
-  echo "Using Node versions: ${NODE_VERSIONS}"
+  echo "Using Node.js versions: ${NODE_VERSIONS}"
 fi
 
 if [ "X${NODE_VERSIONS}" == "X" ]; then
```

Comment:
Possibly change this to ```if [ "X${NODE_VERSIONS}${IOJS_VERSIONS}" == "X" ]; then``` to make it work for either of them without the other if so desired.